app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java

```diff
@@ -238,6 +241,15 @@ protected void addLatestMovies() {
 
     }
 
+    protected void addLatestTVShows() {
```

Comment:
This method can be private. (Not sure why the others aren't, but no need to continue that here :smile:)

Reply:
Okay, will make the others private as well then.